vault/logical_system.go

```diff
@@ -299,6 +300,42 @@ func NewSystemBackend(core *Core, config *logical.BackendConfig) (logical.Backen
 			},
 
 			&framework.Path{
+				Pattern: "lease",
+
+				Fields: map[string]*framework.FieldSchema{
+					"lease_id": &framework.FieldSchema{
+						Type:        framework.TypeString,
+						Description: strings.TrimSpace(sysHelp["lease_id"][0]),
+					},
+				},
+
+				Callbacks: map[logical.Operation]framework.OperationFunc{
+					logical.UpdateOperation: b.handleLease,
```

Comment:
Since `lease_id` is sensitive, we determined it made sense to pass `lease_id` as a parameter with an update operation.  This is currently the preferred method to use in `/sys/renew` and `/sys/revoke`.